client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { CompactCard } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import MaterialIcon from 'calypso/components/material-icon';
+import SectionHeader from 'calypso/components/section-header';
+import Badge from 'calypso/components/badge';
+
+class EmailPlanMailboxesList extends React.Component {
+	render() {
+		const { emails, translate } = this.props;
+
+		if ( ! emails || emails.length < 1 ) {
+			return null;
+		}
+
+		const emailsItems = emails.map( ( email ) => {
+			return (
+				<CompactCard
+					key={ `email-row-${ email.email }` }
+					className="email-plan-mailboxes-list__mailbox-list-item"
+				>
+					<MaterialIcon icon="email" />
+					<span>{ email.email }</span>
+					{ email.isAdmin && <Badge type="info">{ translate( 'Admin' ) }</Badge> }
+				</CompactCard>
+			);
+		} );
+
+		return (
+			<div className="email-plan-mailboxes-list__mailbox-list">
+				<SectionHeader label={ translate( 'Mailboxes' ) } />
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Add New Mailboxes' )` *ES Score: 7*